donation-core/src/main/java/com/donation/crud/impl/UserCrudServiceImpl.java

```diff
@@ -28,6 +28,14 @@ public User add (User user) {
         return user;
     }
 
+    @Override
+    public User update(User user, User newUser) {
+        newUser.setPassword(encoder.encode(newUser.getPassword()));
+        if (newUser.getEmail() == null) newUser.setEmail(user.getEmail());
```

Comment:
please always use {} even if you have only one instruction
if (newUser.getEmail() == null) {
newUser.setEmail(user.getEmail());
}